statsmodels/iolib/foreign.py

```diff
@@ -686,6 +693,10 @@ class StataWriter(object):
                        'd': 8.98846567431158e+307}
     def __init__(self, fname, data, convert_dates=None, encoding="latin-1",
                  byteorder=None):
+        warnings.warn(
+            "StataWriter is deprecated as of 0.10.0 and will be removed in a "
+            "future version.  Use pandas.DataFrame(data).to_stata instead.",
```

Comment:
Same comment: use pandas.DataFrame.to_stata or pandas.io.stata.StataWriter

Also, I think it should be pandas.DataFrame.to_stata, which is the correct function reference.